libafl/src/stages/calibrate.rs

```diff
@@ -327,6 +327,16 @@ where
 
         Ok(())
     }
+
+    fn initialize_progress(&mut self, state: &mut Self::State) -> Result<(), Error> {
```

Comment:
@addisoncrump _technically_ we could just call this from the stage, not sure what's best